tests/periph_spi/main.c

```diff
@@ -57,6 +61,25 @@ static struct {
     spi_cs_t cs;
 } spiconf;
 
+/*
+ * @brief Trigger an update of the scheduler runtime statistics.
+ *
+ * Increases the number of context switches by one as a side effect
+ */
+static void _sched_statistics_trigger(void)
+{
+    sched_statistics_cb(sched_active_pid, sched_active_pid);
+}
+
+static xtimer_ticks64_t _sched_ticks(void)
+{
+    _sched_statistics_trigger();
+    xtimer_ticks64_t runtime_ticks = {
```

Comment:
I think you can just use `xtimer_ticks32_t` here - the measuring period won't be that long and it will be a bit lighter.

Reply:
I agree, it *should* be fine to change this and keeps the test suitable for the 8-bit architectures. Changed in the latest fixup commit.